micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MicronautProcessEngineConfiguration.java

```diff
@@ -29,6 +32,8 @@
 
     private final DatasourceConfiguration datasourceConfiguration;
 
+    private @Inject List<Consumer<ProcessEngineConfiguration>> processEngineConfigurationCustomizers;
```

Comment:
We should not use `@Inject`on a private field because this would require reflection which is not wanted in Micronaut Applications. Please add this field to the constructor instead similar to `datasourceConfiguration`.